README.md

```diff
@@ -884,6 +885,7 @@ Table of Contents
   * [Fontshare](https://www.fontshare.com/) - is a free fonts service. It’s a growing collection of professional grade fonts that are 100% free for personal and commercial use.
   * [Befonts](https://befonts.com/) - Provides several unqiue fonts for personal or commercial use.
   * [Font of web](https://fontofweb.com/) - Identify all the fonts used on a website and how they are used.
+  * [Bunny Fonts](https://fonts.bunny.net/) - Easily integrate Bunny Fonts into your website with a Google Fonts drop-in compatible API. But without the tracking!.
```

Comment:
You can fix the feedback on the other PR here and close that one if you like

Reply:
Like this https://github.com/ripienaar/free-for-dev/pull/2573/commits/716c807521c969e2e9f6f13cf3f9933608dfe3cc
?

Reply:
In which category should I put it?, I ask because the main product of bunny.net is CDN but CDN does not have any free tier.